src/WinRPM.jl

```diff
@@ -50,20 +50,24 @@ if is_unix()
         unsafe_string(x.body), x.http_code
     end
 elseif is_windows()
+   function download(url::AbstractString, filename::AbstractString)
+        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+        tls12 = "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12"
+        client = "New-Object System.Net.Webclient"
+        # in the following we escape ' with '' (see https://ss64.com/ps/syntax-esc.html)
+        downloadfile = "($client).DownloadFile('$(replace(url, "'", "''"))', '$(replace(filename, "'", "''"))')"
+        run(`$ps -NoProfile -Command "$tls12; $downloadfile"`)
+        filename
+    end
+
     function download(source::AbstractString; retry=5)
-        dest = Vector{UInt16}(261)
+        filename = joinpath(@__FILE__, "..", "..", "cache", basename(source))
```

Comment:
`DownloadString` tries to get character encoding using content-type header from the request and not from the response  header. Which basically doesn't help us at all, since in one case we want to download it as utf8 (for parsing the xml) and for the `.rpm` sources we don't want utf8. Right now I have it properly encoding the results for the xml, which is why it doesn't work for downloading the `rpm` sources.